dist.ts

```diff
@@ -0,0 +1,159 @@
+import { Temporal } from "@js-temporal/polyfill";
+import {
+  BASELINE_LOW_TO_HIGH_DURATION,
+  computeBaseline,
+} from "compute-baseline";
+import { Compat, Feature } from "compute-baseline/browser-compat-data";
+import assert from "node:assert/strict";
+import fs from "node:fs";
+import path from "node:path";
+import { fileURLToPath } from "node:url";
+import YAML from "yaml";
+
+/**
+ * Update (or create) a dist YAML file from a feature definition YAML file.
+ *
+ * @param {string} fp The path to the human-authored YAML file.
+ */
+function updateDistFile(fp: string): void {
+  const { dir, name: id } = path.parse(fp);
+  const distPath = path.join(dir, `${id}.dist.yml`);
+
+  const stringify = (y: YAML.Document) => y.toString({ lineWidth: 0 });
+
+  const currentDist = fs.existsSync(distPath)
+    ? YAML.parseDocument(fs.readFileSync(distPath, { encoding: "utf-8" }))
+    : undefined;
+  const newDist = toDist(fp);
+
+  currentDist.commentBefore = undefined; // Suppress the generated comment for comparisons
+
+  // Write a dist only if it's new or modified
+  if (!currentDist || stringify(currentDist) !== stringify(newDist)) {
```

Comment:
Can you save `stringify(newDist)` in a variable to not to it twice?

Reply:
I fixed this alongside dropping the date from comments (which is why I was doing it in the first place) in 7b358fb3d246ede8c98ddd57b2ac96959697256c.